src/Features/VisualBasic/Portable/OrganizeImports/VisualBasicOrganizeImportsService.vb

```diff
@@ -4,16 +4,18 @@ Imports System.Threading
 Imports Microsoft.CodeAnalysis.Host.Mef
 Imports Microsoft.CodeAnalysis.OrganizeImports
 Imports System.Composition
+Imports Microsoft.CodeAnalysis.Editing
 
 Namespace Microsoft.CodeAnalysis.VisualBasic.OrganizeImports
     <ExportLanguageService(GetType(IOrganizeImportsService), LanguageNames.VisualBasic), [Shared]>
     Partial Friend Class VisualBasicOrganizeImportsService
         Implements IOrganizeImportsService
 
         Public Async Function OrganizeImportsAsync(document As Document,
-                                        placeSystemNamespaceFirst As Boolean,
                                         cancellationToken As CancellationToken) As Task(Of Document) Implements IOrganizeImportsService.OrganizeImportsAsync
             Dim root = Await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(False)
+            Dim options = Await document.GetOptionsAsync(cancellationToken).ConfigureAwait(False)
+            Dim placeSystemNamespaceFirst = options.GetOption(GenerationOptions.PlaceSystemNamespaceFirst, document.Project.Language)
```

Comment:
The option is a PerLanguageOption because it can be set differently for C# or VB if you don't use editorconfig